app/views/seriesDetails.scala.html

```diff
@@ -20,8 +20,9 @@ <h1 class="govuk-heading-xl">@Messages("seriesDetails.title")</h1>
             @inputDropdown(
                 seriesDetailsForm("series"),
                 allSeriesData,
-                '_name -> "Series Details",
-                '_label -> Messages("seriesDetails.chooseSeries"),
+                Symbol("_error") -> seriesDetailsForm.errors("error").headOption.getOrElse(None),
```

Comment:
Is this error being used yet? I was trying to work out what type this needs to be, but I can't see any references to it in inputDropdown.scala.html.

Reply:
It is being used but as you suggested I've changed this to `'_error -> seriesDetailsForm.errors("error").headOption,`